pkg/cmd/cli/repomantenance/maintenance.go

```diff
@@ -61,6 +68,10 @@ func (o *Options) Run(f velerocli.Factory) {
 	logger := logging.DefaultLogger(o.LogLevelFlag.Parse(), o.FormatFlag.Parse())
 	logger.SetOutput(os.Stdout)
 
+	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
```

Comment:
Can we use the logger from [above](https://github.com/vmware-tanzu/velero/pull/8580/files#diff-52bcc10aded76b12d6d96b04296ead5d96d75351f65489cdf65ff472f100410bR68) instead of new zap? That way the logger format flags apply here
```
❯ velero server --log-format
completions
--log-format  -- The format for log output. Valid values are text, json.
```
like here https://github.com/vmware-tanzu/velero/pull/8450/files#diff-7a8f33109587a2bd51be299c9d2b3849a38ce9d19715e2ea04449b4602fbba54R242

Reply:
Done